README.md

```diff
@@ -7,6 +7,15 @@
 > If you'd like to learn more about our [tools](https://xenit.eu/open-source), [services](https://xenit.eu/alfresco) and [products](https://xenit.eu/alfresco-products)
 > please visit our [website](https://xenit.eu).
 
+## Important note regarding Care4Alf
```

Comment:
(non-blocking) `Important note` is enough IMHO. Since the README is in the care4alf repo, the fact that is about care4alf is sufficiently implied.